src/density.js

```diff
@@ -46,6 +48,21 @@ export default function() {
         values[x0 + 1 + y0 * n] += xt * (1 - yt) * wi;
         values[x0 + 1 + (y0 + 1) * n] += xt * yt * wi;
         values[x0 + (y0 + 1) * n] += (1 - xt) * yt * wi;
+
+        if(aggregation === 'MEAN') {
+          valueCounts[x0 + y0 * n] += 1;
+          valueCounts[x0 + 1 + y0 * n] += 1;
+          valueCounts[x0 + 1 + (y0 + 1) * n] += 1;
+          valueCounts[x0 + (y0 + 1) * n] += 1;
```

Comment:
Shouldn't the mean be computed with bilinear interpolation too?

```suggestion
          valueCounts[x0 + y0 * n] += (1 - xt) * (1 - yt);
          valueCounts[x0 + 1 + y0 * n] += xt * (1 - yt);
          valueCounts[x0 + 1 + (y0 + 1) * n] += xt * yt;
          valueCounts[x0 + (y0 + 1) * n] += (1 - xt) * yt;
```